consul/kv_consul.go

```diff
@@ -779,6 +800,7 @@ func (kv *consulKV) pairToKvs(
 func (kv *consulKV) renewLockSession(
 	key string,
 	initialTTL string,
+	lockTimeout time.Duration,
```

Comment:
renewLockSession is not using the lockTimeout you have passed as an argument on line 813